sdk/python/pulumi_kubernetes/networking/v1/_inputs.py

```diff
@@ -1078,6 +1006,78 @@ def policy_types(self, value: Optional[pulumi.Input[Sequence[pulumi.Input[str]]]
         pulumi.set(self, "policy_types", value)
 
 
+@pulumi.input_type
+class NetworkPolicyArgs:
```

Comment:
It looks like things are just moved around in Python but it's really hard to tell if there are any significant changes here. Do we trust there are none?

Reply:
Yes, I checked, and it was just reordering some of the classes.